ergotree-ir/src/types/stype_param.rs

```diff
@@ -100,3 +100,14 @@ pub struct STypeParam {
     upper_bound: Option<SType>,
     lower_bound: Option<SType>,
 }
+
+impl STypeParam {
```

Comment:
Don't think this is needed, check comments in sglobal.rs

Reply:
you are right. i removed it.